### ISSUE TITLE
Permission denaied error. using anaconda 3 , windows 8.1 , admin privilage account. 

### ISSUE THREAD

**Issue by shahbrij** [coandco/gtalk_export#7]
_## I am trying to use this code, but keep getting permission denied error. I am not much familier with python , just followed what was given in readme.md. pleasee Help._ 

(C:\ProgramData\Anaconda3) D:\gtalk_export-master>python gtalk_export.py -p D:\g
talk_export-master -j D:\gtalk_export-master -n brij -e _iHadPutMyMailIdCorrectly_ -m MB
OX
Processing mailbox at D:\gtalk_export-master
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\lib\mailbox.py", line 580, in __init__
    f = open(self._path, 'rb+')
**PermissionError: [Errno 13] Permission denied: 'D:\\gtalk_export-master'**

During handling of the above exception, another exception occurred:

**Traceback (most recent call last):
  File "gtalk_export.py", line 154, in <module>
    parse_mailbox(args.mailbox_path, args.name, args.email, args.timestamp_forma
t, args.mbox)
  File "gtalk_export.py", line 52, in parse_mailbox
    mbox = mailbox.mbox(mailbox_path)
  File "C:\ProgramData\Anaconda3\lib\mailbox.py", line 846, in __init__
    _mboxMMDF.__init__(self, path, factory, create)
  File "C:\ProgramData\Anaconda3\lib\mailbox.py", line 588, in __init__
    f = open(self._path, 'rb')
PermissionError: [Errno 13] Permission denied: 'D:\\gtalk_export-master'**

**Comment by coandco** [comment]
"Permission denied" generally means that the script can't access whatever you're trying to point it at.  I don't know that I can really help with that too much, but I did notice that you're trying to pass the same thing for both the JSON file (-j) parameter and the mailbox path (-p) parameter, which doesn't make sense.  A thing can't really be a JSON file and a mailbox path at the same time.

**Comment by shahbrij** [comment]
Sir, I tried to put both of them in diff folder as you suggested .
Also i tried not to use json option at all. I am not giving any timestamp format either.
I also changed my mbox file name to "mailbox.mbox". 

mbox file is in D drive (non os drive), anaconda is in C drive (OS drive) 
I have taken ownership of both the folders to current user which is already administrator. 

Still, I am getting same error.  

Anyother thing i should try. 
Thank you in advance.
